MBBSEmu/CPU/CPUCore.cs

```diff
@@ -2715,6 +2674,20 @@ private void Op_Fsubr()
             FpuStack[Registers.Fpu.GetStackTop()] = result;
         }
 
+        /// <summary>
+        ///     Floating Point Reverse Subtraction (x87)
+        /// </summary>
+        [MethodImpl(CompilerOptimizations)]
+        private void Op_Fsubrp()
+        {
+            var valueToSubtractFrom = GetOperandValueDouble(_currentInstruction.Op0Kind, EnumOperandType.Destination);
+            var valueToSubtract = GetOperandValueDouble(_currentInstruction.Op1Kind, EnumOperandType.Source);
```

Comment:
FSUBR => (DST = SRC - DST)

At least with those EnumOperandTypes, it looks like DST - SRC. I don't really have all the context here, so just double check this is right

Reply:
Yeah -- this was backwards and a little confusing. I fixed the Unit Test to properly reflect subtracting destination (STi/1) from source (ST0). This is how Iced handles the disassembly of FSUBR. Where ST0 has to be the SOURCE operand.